credsweeper/__init__.py

```diff
@@ -25,4 +25,4 @@
 
 CREDSWEEPER_DIR = Path(__file__).resolve().parent
 
-__version__ = "1.4.8"
+__version__ = "1.4.9"
```

Comment:
```suggestion
__version__ = "1.5.0"
```
May be it is a good chance increse the major counter?

Reply:
Okay. After `v1.4.9` let's move up to `v1.5.0`.